setup.py

```diff
@@ -107,8 +107,8 @@ def run(self):
         ],
     entry_points={
         'console_scripts': [
-            'tendrl-gluster-integration = '
-            'tendrl.gluster_integration.manager:main'
+            'tendrl-gluster-integration = tendrl.gluster_integration.manager:main',
+            'vol_utilization = tendrl.gluster_integration.sds_sync.vol_utilization:main'
```

Comment:
Please change the cmd to "tendrl_gluster_vol_utilization" , "vol_utilization" is very ambiguous